### DISCUSSION TITLE
Use Skywalking Webapp UI to query logs

### DISCUSSION THREAD

**Discussion by dresstokill** [apache/skywalking#12465] [Q&A] [answered]
![微信图片_20240722113630](https://github.com/user-attachments/assets/bea0cb7a-3ce1-4d39-b4a3-abb134dddc2d)

I have two springboot applications.
 
elasticsearch-7.15.0
apache-skywalking-apm-bin-9.3.0

As shown in the figure, how to use traceId to query logs of multiple services simultaneously in the Skywalking Webapp UI

**Comment by wu-sheng** [comment, marked as answer]
We don't recommend to do this here. Query trace at trace page, it could link all relative logs from spans.

**Reply by dresstokill** [reply]
![trace-log](https://github.com/user-attachments/assets/2174988b-2e7b-4425-9a80-e5530f996af1)
![picture2](https://github.com/user-attachments/assets/845323b1-d163-45c6-bd51-6e35425469fa)
![picture 3](https://github.com/user-attachments/assets/b292b4d3-d19e-4f03-b9b0-6270591c9b83)

According to what you said, I query trace at trace page, but only one service's log is associated.
As shown in the figure, the endpoint is associated with two services (demo service and sysem service), and when I click the "View Log" button and jump to the log page, only the demo service's logs still be displayed. I have to select the system service in the query criteria to see the logs of the system service.

**Reply by wu-sheng** [reply]
As I said, we prefer span relative logs, or span relative logs, we don't list logs from a distributed system, can't see the point of doing that.

**Reply by dresstokill** [reply]
Thank you for your answer
